ionoscloud/resource_server_vcpu.go

```diff
@@ -0,0 +1,355 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
+)
+
+func resourceVCPUServer() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVCPUServerCreate,
+		ReadContext:   resourceServerRead,
+		UpdateContext: resourceServerUpdate,
+		DeleteContext: resourceServerDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceServerImport,
+		},
+		CustomizeDiff: checkServerImmutableFields,
```

Comment:
This being a new resource it would be best to have these fields with forcenew. This is the idiomatic terraform way of doing things and is easier for the clients if he actually wants to change these fields.

Reply:
@cristiGuranIonos added those in 35a13f8951cffce1c4c25e74f6242197e6cf8354